.env.development_local

```diff
@@ -1,12 +1,5 @@
+# NOTE: Variables need to be prefixed using REACT_APP_:
+# https://create-react-app.dev/docs/adding-custom-environment-variables/
+
 # GATEWAY
 REACT_APP_GATEWAY=http://localhost:8000/api/v1/
-PORT=3000
```

Comment:
Should we restore this? Seems to work without it as well.

Reply:
It's not being used. And if you want to keep it, it should be prefixed by `REACT_APP_`, to be used within the React App.